services/workflows-service/.eslintrc.cjs

```diff
@@ -10,9 +10,8 @@ module.exports = {
     'import/resolver': {
       typescript: {
         alwaysTryTypes: true,
-        project: './tsconfig.json',
+        project: ['**/tsconfig.json'],
```

Comment:
Does this change leave a chance for the wrong tsconfig to be used?

Reply:
I dont think so, from what I read it seems to fit monorepos